### PR TITLE
U instead of STOP codon for TGA

### PR DESCRIPTION
**Pull request by KalinNonchev**

### CODE REVIEW

**Comment by Hoeze** [comment]
@MuhammedHasan can you review this please?

**Comment by KalinNonchev** [comment]
> @MuhammedHasan can you review this please?
> Looks good to me though

I am still awaiting  information how to implement this method: ` _extract_query(self, variant_interval_queryable, sample_id=None)` in SingleSeqProteinVCFSeqExtractor(ProteinVCFSeqExtractor) in [protein.py](https://github.com/KalinNonchev/kipoiseq/blob/KalinNonchev-protein-dl-grch38/kipoiseq/extractors/protein.py) line 279

**Comment by Hoeze** [comment]
@KalinNonchev Did you also include the unit tests?

File: kipoiseq/extractors/protein.py [hunk 1]
```diff
@@ -103,25 +143,47 @@ def __len__(self):
...
     def __getitem__(self, idx):
         return self.get_seq(self.transcripts[idx])
 
     def overlaped_cds(self, variants):
+        """
+        19.03.20
+        This method is implemented in the ProteinVCFSeqExtractor as
+        extract_all()
+        
+        The infromation below is old.
```
Comment (Hoeze) [inline kipoiseq/extractors/protein.py]: information

**Comment by KalinNonchev** [comment]
@MuhammedHasan can you review my code? Thank you

**Pull request closed without merging**